README.md

```diff
@@ -2,7 +2,7 @@
 
 A Ruby library for harvesting metadata from [OAI-PMH](https://www.openarchives.org/OAI/openarchivesprotocol.html) repositories.
 
-**Current version:** 0.9.0  
+**Current version:** 0.9.0
```

Comment:
The trailing whitespace is significant here (it inserts a Markdown linebreak rather than a new paragraph), can we restore it?

Reply:
oh this is because my sublime settings